using_images/other_images/jenkins.adoc

```diff
@@ -52,9 +52,24 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
 
 === Initializing Jenkins
 
+ifdef::openshift-origin[]
+There are two methods for managing authentication with the {product-title} Jenkins images when they are initially started.
+
+* Default Jenkins authentication
+
+* {product-title} OAuth authentication provided by the OpenShift Login plugin.
+
+==== Default Jenkins authentication
+
+Default Jenkins authentication will be used when the {product-title} Jenkins images are first launched outside of an {product-title} pod,
+or when the `*OPENSHIFT_ENABLE_OAUTH*` environment variable is either missing or set to `false` when running inside a pod.
+
+endif::openshift-origin[]
+
 The first time you start Jenkins, the configuration is created along with the administrator user and password.
-The default login is `*admin/password*`.
-The default password can be configured by setting the `*JENKINS_PASSWORD*` environment variable.
+The default user login is `*admin/password*` if running outside an {product-title} pod.
```

Comment:
As part of getting rid of the ifdef's, I'm getting rid of any mention of oauth in the default jenkins auth section.  The openshift oauth section explains the default means for logging in for that config.